packages/contract/src/pay/DaimoPayRelayer.sol

```diff
@@ -124,7 +167,7 @@ contract DaimoPayRelayer is Ownable2Step {
         Call[] calldata startCalls,
         bytes calldata bridgeExtraData,
         Call[] calldata postCalls
-    ) public payable onlyOwner {
+    ) public payable onlyRole(RELAYER_ROLE) {
```

Comment:
it might be worth renaming this from `RELAYER_ROLE` / `relayer` etc to `RELAY_EOA_ROLE` / `relayEOA` or similar, to distinguish it from the relayer contract